express/blocks/floating-button/floating-button.js

```diff
@@ -272,6 +272,7 @@ function buildTools($wrapper, $tools, delayInSeconds = 3) {
   const $cta = $floatingButton.querySelector('a');
   const $toggleButton = createTag('a', { class: 'toggle-button' });
   const $toggleIcon = getIconElement('plus-icon-22');
+  const $circleAnimation = getLottie('plus-animation', '/express/icons/plus-animation.json');
```

Comment:
@qiyundai can we load this at the last possible moment to decrease the impact on LHS?

Reply:
Sure! I pushed it down a few lines to right before the element is being injected on the page.